docs/src/pages/backend-adapters/fetch.mdx

```diff
@@ -129,6 +130,60 @@ app.route("/api/uploadthing", ut);
 export default app;
 ```
 
+### Cloudflare Workers
+
+```ts filename="src/worker.ts"
+import { createServerHandler } from "uploadthing/server";
+
+import { uploadRouter } from "./uploadthing.ts";
+
+export default {
+  async fetch(request: Request, env: Env, ctx: ExecutionContext) {
+    const handlers = createServerHandler({
+      router: uploadRouter,
+      config: {
+        /**
+         * Since workers doesn't have envs on `process`. We need to pass
+         * secret and isDev flag manually.
+         */
+        uploadthingSecret: env.UPLOADTHING_SECRET,
+        isDev: env.MODE === "development",
```

Comment:
Can we detect this with import `isDevelopment` from std-env?

Reply:
added the isDevelopment check but we still need this explicit for CF workers due to the variable not being bound to the global scope